examples/maze-solver/maze_tests.py

```diff
@@ -22,7 +22,7 @@
 
 
 def fill_with_zeros(solution_dict, n_rows, n_cols, ignore_list=None):
-    keys = solution_dict.keys() if ignore_list is None else list(solution_dict.keys()) + ignore_list
```

Comment:
From the code below, looks like it would be better (faster lookups) to make `keys` a `set`, not `list`.